### PR TITLE
Fix shacl_validate mutating`self._rdf`

### LINKED ISSUE #122: Remove rdfProperty, rdfs:Resource, and rdfs:subPropertyOf predicates
For some reason we have all these `rdfProperty`, `rdfs:Resource`, `rdfs:subPropertyOf` predicates in the FairWorkflow rdf:
```
pwo:hasFirstStep a rdf:Property,
            rdfs:Resource ;
        rdfs:subPropertyOf pwo:hasFirstStep .

    dul:precedes a rdf:Property,
            rdfs:Resource ;
        rdfs:subPropertyOf dul:precedes .

    rdf:type a rdf:Property,
            rdfs:Resource ;
        rdfs:subPropertyOf rdf:type .

    rdfs:subPropertyOf a rdf:Property,
            rdfs:Resource ;
        rdfs:subPropertyOf rdfs:subPropertyOf .
```
See: https://github.com/fair-workflows/fairworkflows/blob/main/examples/test_fairworkflows.ipynb

They were not there before, so some commit must have introduced them. We should adapt the code that adds this (not add more code that removes this)

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #122 
This was fixed by https://github.com/RDFLib/pySHACL/issues/69 on my request 😄

### CODE REVIEW

File: fairworkflows/rdf_wrapper.py [hunk 1]
```diff
@@ -101,9 +102,7 @@ def remove_attribute(self, predicate, object=None):
     def shacl_validate(self):
         sg = rdflib.Graph()
         sg.parse(PLEX_SHAPES_SHACL_FILEPATH, format='ttl')
-        conforms, results_graph, results_text = pyshacl.validate(
```
Comment (svenvanderburg) [inline fairworkflows/rdf_wrapper.py:104]: These are all default options, so I removed them.

File: requirements.txt [hunk 2]
```diff
@@ -4,4 +4,4 @@ pytest
 pyyaml
 rdflib<6.0.0,>=5.0.0
 requests
-pyshacl
+pyshacl>=0.14.1
```
Comment (svenvanderburg) [inline requirements.txt:7]: This is the actual fix

File: fairworkflows/rdf_wrapper.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import copy
```
Comment (raar1) [inline fairworkflows/rdf_wrapper.py:1]: Is copy still needed now that pyshacl no longer mutates the graph?

**Review by raar1 — APPROVED** [review]

File: fairworkflows/rdf_wrapper.py (hunk 3 above)
Comment (svenvanderburg) [reply fairworkflows/rdf_wrapper.py]: Nope, I removed it!

**Pull request merged**